src/components/Comments/hooks/useCommentAction.js

```diff
@@ -40,14 +44,13 @@ const useCommentAction = () => {
         }
       }
     } catch (e) {
-      if (__DEV__) {
-        console.log(e, 'eman');
-      }
+      console.log(e, 'eman');
     }
   };
 
   return {
-    showAlertDelete
+    showAlertDelete,
+    onDeleteCommentClicked
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The return object now includes `onDeleteCommentClicked`. Ensure that all places where this hook is used are updated to handle this additional returned value.


<!-- This is an auto-generated comment by OSS CodeRabbit -->